sensitivity_games/point_mass.py

```diff
@@ -89,13 +89,13 @@ def modify_B(self):
                         [0, 0],
                         [0, self.dt/m]])
 
-    def visualize(self, X, U, goal, block):
+    def visualize(self, X, U, xf, block):
         '''
         '''
-        xmax = max([x[0].detach() for x in X])
-        xmin = min([x[0].detach() for x in X])
-        ymax = max([x[2].detach() for x in X])
-        ymin = min([x[2].detach() for x in X])
+        xmax = max(max([x[0].detach() for x in X]), xf[0])
```

Comment:
wait don't you want to look at error somewhere? I'm confused why there are two maxes here

Reply:
this is just used for plotting, so i can set the axes correctly.